<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-601D2VQ" failures="1" tests="1" name="Default test" time="4.387" errors="0" timestamp="2024-02-29T20:54:02 EET">
  <testcase classname="automationteststore.TestStore$TestArbi" name="sumoftheprices" time="4.134">
    <failure type="java.lang.NumberFormatException" message="empty String">
      <![CDATA[java.lang.NumberFormatException: empty String
at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1842)
at java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
at java.base/java.lang.Double.parseDouble(Double.java:651)
at automationteststore.TestStore$TestArbi.sumoftheprices(TestStore.java:189)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- sumoftheprices -->
  <testcase classname="automationteststore.TestStore$TestArbi" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
  <testcase classname="automationteststore.TestStore$TestArbi" name="contactnumberinthefooter" time="0.0">
    <ignored/>
  </testcase> <!-- contactnumberinthefooter -->
  <testcase classname="automationteststore.TestStore$TestArbi" name="haircaresection" time="0.0">
    <ignored/>
  </testcase> <!-- haircaresection -->
  <testcase classname="automationteststore.TestStore$TestArbi" name="addallitems" time="0.0">
    <ignored/>
  </testcase> <!-- addallitems -->
  <testcase classname="automationteststore.TestStore$TestArbi" name="signup" time="0.0">
    <ignored/>
  </testcase> <!-- signup -->
  <testcase classname="automationteststore.TestStore$TestArbi" name="printalltheitems" time="0.0">
    <ignored/>
  </testcase> <!-- printalltheitems -->
</testsuite> <!-- Default test -->
